### ISSUE TITLE
systemd user services - enhancement.

### ISSUE THREAD

**Issue by seungjin** [gwbres/systemctl#28]
It would be awesome if it support '--user' systemd user services! 
`systemctl --user status foo.service`

**Comment by kimtore** [comment]
Agreed; I'm working on a little project right now that needs to manage user services and could really use this feature!

**Comment by BlakeGardner** [comment]
Came here looking for the same thing, I have need to manage / enumerate user specific units.
